charts/ceph-csi-rbd/values.yaml

```diff
@@ -144,6 +144,8 @@ provisioner:
   deployController: true
   # Timeout for waiting for creation or deletion of a volume
   timeout: 60s
+  # cluster name to set on the RBD image
+  clustername: "k8s-cluster-1"
```

Comment:
let's not add this to default deployment as it does not make sense to set dummy clusterName, comment this out.